src/app/containers/RadioSchedule/utilities/processRadioSchedule.js

```diff
@@ -51,6 +68,19 @@ export default (data, service, currentTime) => {
         publishedTimeDuration,
       } = program;
 
+      const brandTitle = path(['brand', 'title'], program);
+      const episodeTitle = path(['episode', 'presentationTitle'], program);
+
+      if (!publishedTimeStart) {
+        logMissingFieldError('publishedTimeStart');
+      }
+      if (!brandTitle) {
+        logMissingFieldError('title');
+      }
+      if (!episodeTitle) {
+        logMissingFieldError('presentationTitle');
```

Comment:
I think we are interested in when we are unable to render any schedules as well, when `scheduleDataIsComplete` is false